Assets/Scripts/Screeps3D/Effects/AttackEffect.cs

```diff
@@ -13,10 +13,14 @@ public class AttackEffect : MonoBehaviour
         private const float _attackDuration = 2;
         private Vector3 _target;
         
-        public void Load(Vector3 target)
+        public void Load(Vector3 target, Color? color)
         {
             _target = target;
             _time = 0f;
+            if(color != null) {
+                ParticleSystem.MainModule settings = _attackParticles.main;
+                settings.startColor = Color.blue;
```

Comment:
Should this not be the color variable instead of blue? does not seem like the color argument is used?

Reply:
for now it's hardcoded to see if it works (it does not - i need to change trail color)